pyiron_base/generic/hdfio.py

```diff
@@ -291,40 +291,60 @@ def copy_to(self, destination, file_name=None, maintain_name=True):
             FileHDFio: FileHDFio object pointing to a file which now contains the same content as file of the current
                        FileHDFio object.
         """
+        def _internal_copy(source, source_path, target, target_path, maintain_flag):
+            """
+            Internal function to copy content of one HDF5 file to another or copy a group within the same HDF5 file. 
+            
+            Args:
+                source (h5py.File): HDF5 File object 
+                source_path (str): Path inside the source HDF5 file 
+                target (h5py.File): HDF5 File object
+                target_path (str): Path inside the target HDF5 file 
+                maintain_flag (bool): Maintain the same group name 
+            """
+            if maintain_flag:
+                try:
+                    target.create_group(target_path)
+                except ValueError:
+                    pass  # In case the copy_to() function failed previously and the group already exists. 
+            if target_path == "/":
+                source.copy(h5_path, "/") if source == target else source.copy(h5_path, target)
+                    source.copy(source_path, "/")
```

Comment:
I don't know how to do suggestions over multiple lines yet, but the rest of this `if target_path` clause will break now that we one-lined it

Reply:
Fixed